tests/test_runner/test_debug_sql.py

```diff
@@ -61,28 +61,14 @@ def test_output_verbose(self):
         for output in self.verbose_expected_outputs:
             self.assertIn(output, full_output)
 
-    if six.PY3:
-        expected_outputs = [
-            ('''QUERY = 'SELECT COUNT(%s) AS "__count" '''
-                '''FROM "test_runner_person" WHERE '''
-                '''"test_runner_person"."first_name" = %s' '''
-                '''- PARAMS = ('*', 'error');'''),
-            ('''QUERY = 'SELECT COUNT(%s) AS "__count" '''
-                '''FROM "test_runner_person" WHERE '''
-                '''"test_runner_person"."first_name" = %s' '''
-                '''- PARAMS = ('*', 'fail');'''),
-        ]
-    else:
-        expected_outputs = [
-            ('''QUERY = u'SELECT COUNT(%s) AS "__count" '''
-                '''FROM "test_runner_person" WHERE '''
-                '''"test_runner_person"."first_name" = %s' '''
-                '''- PARAMS = (u'*', u'error');'''),
-            ('''QUERY = u'SELECT COUNT(%s) AS "__count" '''
-                '''FROM "test_runner_person" WHERE '''
-                '''"test_runner_person"."first_name" = %s' '''
-                '''- PARAMS = (u'*', u'fail');'''),
-        ]
+    expected_outputs = [
+        '''SELECT COUNT('*') AS "__count" '''
```

Comment:
Meanwhile, I merged a conflicting patch. I think you just need to remove the quotes from around the star.